### PR TITLE
Worked a little.

### PR DESCRIPTION
**Pull request by Youngestdev**
Fixed some errors and added  basic updates.

### CODE REVIEW

**Review by sdmg15 — COMMENTED** [review]
They're some conflicts.

**Comment by Youngestdev** [comment]
Where @sdmg15 ?

**Comment by sdmg15** [comment]
Look at home.js/Register.js files .
Here I have compile error.

**Comment by Youngestdev** [comment]
Talking about the Navbar ?

File: src/Components/Auth/Login.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Comment (sdmg15) [inline src/Components/Auth/Login.js:1]: I'm talking about this.

**Pull request merged**